api/types/backend/backend.go

```diff
@@ -26,6 +26,16 @@ type ContainerAttachConfig struct {
 	MuxStreams bool
 }
 
+// LogMetaData provides meta data for a partial log message. Messages exceeding a predefined size
+// are split into chunks with this metadata. The expectation is for the logger endpoints to assemble
+// the chunks using this metadata.
+type LogMetaData struct {
+	IsPartial   bool
```

Comment:
Can we document these fields?

```
ID string // identifies the log message or group of messages comprising a single record 
Ordinal // ordering of message in partial group
IsPartial bool // true if this message is made up of 2 or more messages 
LastPartial bool // true if this message is last of a partial
```

Also, we might be able to get rid of the need for the partial bool if we use 0 for complete message and 1 for first ordinal of partial, then all you need is `last` bool.

Reply:
Doc-ed and removed `FirstPartial`